girder-dandi-archive/tests/rest/test_dandiset_ownership.py

```diff
@@ -0,0 +1,88 @@
+import json
+
+import pytest
+
+from pytest_girder.assertions import assertStatusOk
+
+pytestmark = pytest.mark.plugin("dandi_archive")
+
+
+@pytest.fixture
+def admin_created_dandiset(server, admin):
+    resp = server.request(
+        path="/dandi",
+        method="POST",
+        user=admin,
+        params={
+            "name": "admin_created_dandiset",
+            "description": "Admin created this test dandiset.",
+        },
+    )
+    assertStatusOk(resp)
+    return resp.json
+
+
+@pytest.fixture
+def multi_owner_dandiset(server, admin, user, admin_created_dandiset):
+    request_body = json.dumps([user], default=str)
+    identifier = admin_created_dandiset["meta"]["dandiset"]["identifier"]
+
+    assertStatusOk(
+        server.request(
+            path=f"/dandi/{identifier}/owners",
+            method="PUT",
+            body=request_body,
+            user=admin,
+            type="application/json",
+        )
+    )
+
+    return admin_created_dandiset
+
+
+def test_initial_dandiset_owners(server, admin_created_dandiset):
+    """Assert that the list of owners isn't zero.
+
+    The group members should always contain at least the dandiset creator.
+    """
+    identifier = admin_created_dandiset["meta"]["dandiset"]["identifier"]
+    resp = server.request(path=f"/dandi/{identifier}/owners", method="GET")
+
+    assertStatusOk(resp)
+    assert len(resp.json) == 1
```

Comment:
I think it's worth asserting something about the one user, even if it's just an ID.